torax/_src/plotting/plotruns_lib_test.py

```diff
@@ -18,38 +18,77 @@
 
 from absl.testing import absltest
 from absl.testing import parameterized
+import matplotlib.pyplot as plt
+from matplotlib.figure import Figure
 from torax._src import path_utils
 from torax._src.config import config_loader
 from torax._src.plotting import plotruns_lib
 from torax._src.test_utils import paths
 
 
 class PlotrunsLibTest(parameterized.TestCase):
+  skip_data_files_list = [
+  ]
```

Comment:
Move onto one line and also make a tuple such that it cannot be changed

Also add the type

Reply:
It feels weird to allow list manipulation as then depending what order the tests are executed in or where they execute could alter what happens here

But as commented I would prefer this gets removed